ffmeta/scripts/populate_tables.py

```diff
@@ -110,6 +110,11 @@ def populate_tables(quiet=False):
                     distinct_subtopics.add(s)
 
         in_FFC_file = row['in_FFC_file']
+        obs = row['obs']
+        min = row['min']
```

Comment:
for `min` and `max`, you may want to use `min_` and `max_` instead (a popular convention when reusing built-in function names in python.